libbpf-tools/memleak.c

```diff
@@ -1041,8 +1054,8 @@ int attach_uprobes(struct memleak_bpf *skel)
 	ATTACH_UPROBE_CHECKED(skel, realloc, realloc_enter);
 	ATTACH_URETPROBE_CHECKED(skel, realloc, realloc_exit);
 
-	ATTACH_UPROBE_CHECKED(skel, mmap, mmap_enter);
-	ATTACH_URETPROBE_CHECKED(skel, mmap, mmap_exit);
+	ATTACH_UPROBE(skel, mmap, mmap_enter);
+	ATTACH_URETPROBE(skel, mmap, mmap_exit);
 
```

Comment:
Why do you omit the check here ? Please mention that in commit message.
If mmap/munmap does not exist in libjemalloc.so, maybe we can check the specific errno and skip them.

Reply:
i add some comments, errno skip added in ATTACH_URETPROBE_CHECKED may influence other api, so just remove like memleak.py may be better?

Reply:
Maybe:
```c
if (symbol_prefix)
	ATTACH_UPROBE
else
	ATTACH_URETPROBE_CHECKED
```
and some inline comments.

Reply:
> Maybe:
> 
> ```c
> if (symbol_prefix)
> 	ATTACH_UPROBE
> else
> 	ATTACH_URETPROBE_CHECKED
> ```
> 
> and some inline comments.

done!

Reply:
@chenhengqi Check it out when you have time，all comments finished. Thank you.